Source/DafnyCore/Auditor/Assumption.cs

```diff
@@ -4,56 +4,90 @@
 
 namespace Microsoft.Dafny.Auditor;
 
-public record AssumptionDescription(string issue, string mitigation, bool isExplicit) {
+public record AssumptionDescription(
+  string issue,
+  string mitigation,
+  string mitigationIETF,
+  bool isExplicit
+) {
   public static AssumptionDescription HasAxiomAttribute = new(
     issue: "Declaration has explicit [{:axiom}] attribute.",
     mitigation: "Provide a proof or test.",
+    mitigationIETF: "SHOULD provide a proof or test.",
     isExplicit: true);
   public static AssumptionDescription HasVerifyFalseAttribute = new(
     issue: "Declaration has [{:verify false}] attribute.",
     mitigation: "Remove and prove if possible.",
+    mitigationIETF: "MUST remove and prove.",
```

Comment:
replace by assertion and prove?

Reply:
I meant remove the _attribute_. I should update the wording to clarify that!